components/videoModal.tsx

```diff
@@ -72,7 +73,12 @@ export const VideoModal = ({
   return (
     <div
       className={classNames(
-        "h-full rounded",
+        "h-full",
+        roundedEdges === null ||
+          roundedEdges === undefined ||
+          roundedEdges === true
+          ? "rounded"
+          : "rounded-none",
```

Comment:
```suggestion
        roundedEdges !== false ? "rounded": "rounded-none",
```

can we add this to improve readability?